programs/mango-v4/src/instructions/token_liq_with_token.rs

```diff
@@ -46,7 +46,7 @@ pub fn token_liq_with_token(
     let liqee_liq_end_health = liqee_health_cache.health(HealthType::LiquidationEnd);
     liqee_health_cache.require_after_phase1_liquidation()?;
 
-    if !liqee.check_liquidatable(&liqee_health_cache)? {
+    if liqee.check_liquidatable(&liqee_health_cache)? == CheckLiquidatable::NotLiquidatable {
```

Comment:
This (and check other uses too) changes the behavior.
- Liquidatable: continue on (good)
- NotLiquidatable: previously errored the instruction, now silently returns (arguably ok, but should at least log?)
- BecameNotLiquidatable: previously early-exited the instruction to commit the being_liquidated=false change, now goes into the liquidation code and probably errors there

Suggestion: `Change this to != CL::Liquidatable` and add a log for NotLiquidatable in check_liquidatable()